add ini option to disable string escape for parametrization (#4995)
pytest-dev/pytest: doc/en/parametrize.rst

    test_expectation.py:8: AssertionError
    ==================== 1 failed, 2 passed in 0.12 seconds ====================

.. note::

    pytest by default escapes any non-ascii characters used in unicode strings
    for the parametrization because it has several downsides.

>     If however you would like to use unicode strings in parametrization, use this option in your ``pytest.ini``:

Reviewer: ```suggestion
    If however you would like to use unicode strings in parametrization and see them in the terminal as is (non-escaped), use this option in your ``pytest.ini``:
```

    .. code-block:: ini

        [pytest]
        disable_test_id_escaping_and_forfeit_all_rights_to_community_support = True

>     to disable this behavior, but keep in mind that this might cause unwanted side effects and

Reviewer: ```suggestion
    Keep in mind however that this might cause unwanted side effects and
```
